### DISCUSSION TITLE
Different flow for first login

### DISCUSSION THREAD

**Discussion by ilevac** [goauthentik/authentik#10251] [General]
Hi,
I'm new using Authentik.
It's currently setup with an AD and it work perfectly.

I'm looking for a way to have a different workflow for the first login on a new device.
As of now to access certain service, there is the login/pass from AD + TOTP + FIDO.
Having two 2FA is kind of painfull so I would like to require the 2 only the first login on a new device and then have only 1 required.

Is it something doable?

R,
Jo

**Comment by tmassimi** [comment]
I have the same problem, have you found some way?

I'm checking if it is possible to have a passwordless login and then run a flow from API for the second login

**Comment by tmassimi** [comment]
...instead of run a flow from API, it is possible to run a flow from the /if/your_flow interface.

so: first time login with strong authentication, next logins will be passwordless

second login will be required from your application only if there is no current  (php) session active for that user in your application
